device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/Nokia-IXR7250E-36x100G/0/port_config.ini

```diff
@@ -0,0 +1,21 @@
+# name         lanes                    alias            index  asic_port_name role  speed    coreid  coreportid  numvoq
+Ethernet0      72,73,74,75              Ethernet0        1      Eth0-ASIC0     Ext   100000   1       1           8
+Ethernet1      80,81,82,83              Ethernet1        2      Eth1-ASIC0     Ext   100000   1       2           8
+Ethernet2      88,89,90,91              Ethernet2        3      Eth2-ASIC0     Ext   100000   1       3           8
+Ethernet3      96,97,98,99              Ethernet3        4      Eth3-ASIC0     Ext   100000   1       4           8
+Ethernet4      104,105,106,107          Ethernet4        5      Eth4-ASIC0     Ext   100000   1       5           8
+Ethernet5      112,113,114,115          Ethernet5        6      Eth5-ASIC0     Ext   100000   1       6           8
+Ethernet6      120,121,122,123          Ethernet6        7      Eth6-ASIC0     Ext   100000   1       7           8
+Ethernet7      128,129,130,131          Ethernet7        8      Eth7-ASIC0     Ext   100000   1       8           8
```

Comment:
Can we change it in a similar fasion for 400G sku also - Nokia-IXR7250E-36x400G

Reply:
Based on the conversation in the weekly team meeting, we will address this on a separated PR